src/VscPreviewExtension/DfmParse/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.dll
+*.exe
+*.config
```

Comment:
you can create a empty folder named DfmParse before you copy files into this folder